marshaller.go

```diff
@@ -131,10 +152,23 @@ func constructLikeClause(v interface{}, fieldName string, exclude bool) string {
 
 func buildInClause(v interface{}, fieldName string) string {
 	var buff strings.Builder
-	items, ok := v.([]string)
-	if !ok {
+	var items []string
+	inSingleQuotes := false
```

Comment:
May be rename the variable to `useSingleQuotes`? Just makes it more descriptive

Reply:
Sure